Core/Core/Serialisation/BaseObjectSerializerV2.cs

```diff
@@ -217,7 +216,7 @@ public string Serialize(Base baseObj)
     IEnumerable<string> dynamicProperties = baseObj.GetDynamicMembers();
 
     // propertyName -> (originalValue, isDetachable, isChunkable, chunkSize)
```

Comment:
So object has to be nullable because GetValue() can return null presuambly and we add the tuple anyways, even if baseValue is null? If GetValue() can return null are we sure that Name is always valid? Again, appreciate this is slighly beyond the scope of the nullability change

Reply:
Similar occurs below for dynamicProps also

Reply:
We allow for `null` dynamic values.
I wanted to make sure the nullability syntax was correct here